Homebrew/Resolute Spellbook/firstdaybreak-additions.json

```diff
@@ -0,0 +1,370 @@
+                                    "name": "Residual Electricity - Attack",
+                                    "automation": [
+                                        {
+                                            "type": "target",
+                                            "target": "all",
+                                            "effects": [
+                                                {
+                                                    "type": "ieffect2",
+                                                    "name": "Residual Electricity - Effect",
+                                                    "duration": 1,
+                                                    "desc": "Can't make attacks, no reactions"
+                                                },
+                                                {
+                                                    "type": "damage",
+                                                    "damage": "1d6 [lightning]",
+                                                    "higher": {
+                                                        "3": "1d6",
+                                                        "4": "2d6",
+                                                        "5": "3d6",
+                                                        "6": "4d6",
+                                                        "7": "5d6",
+                                                        "8": "6d6",
+                                                        "9": "7d6"
+                                                    }
+                                                }
+                                            ]
+                                        }
+                                    ],
+                                    "activation_type": 2
+                                }
+                            }
+                        ],
+                        "buttons": [
+                            {
+                                "automation": [
+                                    {
+                                        "type": "remove_ieffect",
+                                        "removeParent": "always"
+                                    }
+                                ],
+                                "label": "Residual Electricity - End",
+                                "style": 4
+                            }
+                        ],
+                        "end": true
+                    }
+                ]
+            },
+            {
+                "type": "target",
+                "target": "all",
+                "effects": [
+                    {
+                        "type": "save",
+                        "stat": "dex",
+                        "fail": [
+                            {
+                                "type": "damage",
+                                "damage": "{damage} [lightning]"
+                            }

```

Comment:
For subsequent saves using {damage} we also want to add fixedValue: true as well. This saves damage bonuses from -d parameters from applying twice (once to the roll node and again to the damage)

Reply:
If referencing a roll automation above the damage node should have "fixedValue": true in it to prevent any `-d` parameters from applying twice